src/librustc_codegen_ssa/traits/intrinsic.rs

```diff
@@ -8,13 +8,17 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-use super::Backend;
 use super::HasCodegen;
 use mir::operand::OperandRef;
 use rustc::ty::Ty;
 use rustc_target::abi::call::FnType;
 use syntax_pos::Span;
 
+#[derive(Copy, Clone)]
+pub enum OverflowOp {
+    Add, Sub, Mul
+}
```

Comment:
@denismerigoux put some enums in `common.rs` but I'm not sure, this might be better.